pom.xml

```diff
@@ -143,11 +137,6 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>junit</artifactId>
-      <scope>test</scope>
-    </dependency>
```

Comment:
FTR, added in https://github.com/jenkinsci/branch-api-plugin/pull/84 for no apparent reason.